pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -149,15 +150,16 @@ func Ordering() []string {
 	return predicatesOrdering
 }
 
-// FitPredicate is a function that indicates if a pod fits into an existing node.
-// The failure information is given by the error.
-type FitPredicate func(pod *v1.Pod, meta PredicateMetadata, nodeInfo *schedulernodeinfo.NodeInfo) (bool, []PredicateFailureReason, error)
-
 // NodeInfo interface represents anything that can get node object from node name.
+// TODO(ahg-g): should be deleted, still exist because kubelet depends on it.
 type NodeInfo interface {
 	GetNodeInfo(nodeName string) (*v1.Node, error)
 }
 
+// FitPredicate is a function that indicates if a pod fits into an existing node.
+// The failure information is given by the error.
+type FitPredicate func(pod *v1.Pod, meta PredicateMetadata, nodeInfo *schedulernodeinfo.NodeInfo) (bool, []PredicateFailureReason, error)
+
```

Comment:
Move it back to its original position? so that it doesn't generate a diff.

Reply:
Oh, that's because I added NodeInfo back not in the same place after I realized that kubelet uses it, anyway we will remove NodeInfo soon, so this should be fine.